CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v4.4.2
+* Fixed: If you had access an element but none of the extended data columns you could still match on the extended data row using a query with no filters. This would result in the row being counted in `totalHits`, but it could not be retrieved from Accumulo.
```

Comment:
Does this require a reindex? If so lets bump it to at least 4.5

Reply:
It does. Fixed with: 575b8a8b960d6b77f8828e2c85bf1347503651c6